test/01_basic/check.py

```diff
@@ -0,0 +1,9 @@
+import glob
+import subprocess
+import sys
+
+build_identifiers = subprocess.check_output([sys.executable, '-m', 'cibuildwheel', '--print-build-identifiers'], universal_newlines=True).strip().split('\n')
```

Comment:
Hmm. Maybe on reflection, just one test needs to check that the build identifiers match the produced wheels. Then we can trust it after that.

Reply:
> I'm not sure on getting cibuildwheel to tell us what it should be producing... a mistake in the python configurations would then not be caught by the tests.

Good point; the using this `--print-build-identifiers` still has turned out to be more code duplication than expected. It would be good if we could make some utilities Python code to be shared by the different tests.

> The --print-build-identifiers arg will be useful regardless :).

If we can do the previous, should I then split up this PR into two separate ones? One for the tests, and one for `--print-build-identifiers`?

Reply:
> Maybe on reflection, just one test needs to check that the build identifiers match the produced wheels. Then we can trust it after that.

Problem here is that there's no unambiguous mapping from build identifier to wheel name. Particularly in the `cp27-cp27m` and `cp27-cp27mu` cases, which both have `cp27-...` identifiers.

Reply:
> Problem here is that there's no unambiguous mapping from build identifier to wheel name. Particularly in the cp27-cp27m and cp27-cp27mu cases, which both have cp27-... identifiers

argh, good point. maybe just statically writing out expected wheels in the test will be better then.

> If we can do the previous, should I then split up this PR into two separate ones? One for the tests, and one for --print-build-identifiers?

I think for the later tests might find the build_identifiers useful, let's keep it in for now